src/com/boxboat/jenkins/pipeline/promote/BoxPromote.groovy

```diff
@@ -33,12 +35,15 @@ class BoxPromote extends BoxBase<PromoteConfig> implements Serializable {
     @Override
     def init() {
         super.init()
+
         if (!config.promotionKey) {
             // abort, since pipeline may refresh without any parameters
             Config.pipeline.currentBuild.result = 'ABORTED'
             Config.pipeline.error "'config.promotionKey' must be set"
+        } else {
```

Comment:
else statement is redundant; `Config.pipeline.error` throws in if block

Reply:
Oops, think the original direction had more conditions here. Thanks!